internal/commands/build_test.go

```diff
@@ -126,6 +126,19 @@ func testBuildCommand(t *testing.T, when spec.G, it spec.S) {
 					h.AssertContains(t, outBuf.String(), "Builder 'heroku/builder:24' is trusted")
 				})
 			})
+
+			when("the builder is not trusted", func() {
```

Comment:
Thank you for adding a test, however I don't believe the test changes mean the bug is now tested.

There are a number of different scenarios:
1. The builder in a built-in known builder that is both trusted and suggested
2. The builder in a built-in known builder that is trusted but not suggested
3. The builder has been manually trusted via the user provided config
4. The builder is untrusted

The existing tests in this file appear to test (1) + (3).

The new test you have added tests (4).

However, the bug was in the behaviour for (2), and there isn't a test for that.

Reply:
Added test for (2)